frame/babe/src/lib.rs

```diff
@@ -195,6 +195,25 @@ decl_module! {
 }
 
 impl<T: Trait> RandomnessT<<T as frame_system::Trait>::Hash> for Module<T> {
+	/// Some BABE blocks have VRF outputs where the block producer has exactly one bit of influence,
+	/// either they make the block or they do not make the block and thus someone else makes the
+	/// next block. Yet, this randomness is not fresh in all BABE blocks.
```

Comment:
All BABE blocks need fresh VRF outputs for randomness.  Initially, the extra AURA-BABE blocks lacked these fresh VRF outputs, but parachains depend upon the, so they must be added.

Reply:
This is still the case as #5501 has not been merged yet. We should update these docs on that PR to reflect the changes.